test/utils.py

```diff
@@ -77,7 +83,8 @@ def cibuildwheel_run(project_path, package_dir=".", env=None, add_env=None, outp
                 "--output-dir",
                 str(output_dir or tmp_output_dir),
                 str(package_dir),
-            ],
+            ]
+            + add_args,
```

Comment:
A very minor thing, but it would fit the style of the codebase a bit more to do `*add_args` inside the list.